### PR TITLE
Bluetooth UI

### PR DESCRIPTION
**Pull request by thedetukov**
### **Description**  ### 
____________________________________________________________________________________________________  
  ###  What does this PR do?  ### 
- This Pull Request includes implementation of UI part of Bluetooth popup window.
In future, the window will be open from icon click in operation system's menu bar.

- We trying to do UI as close as possible to Mac OS.
For example see MacOS experience youtube video https://youtu.be/Z1HUsot3fM8?t=12

- Current implementation is read-only (except On/Off) based on Mock data.
So you may just see list of devices with connection and battery level information.

image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -48,5 +48,6 @@
 	"cSpell.words": [
 		"Cupertino",
 		"switchtile"
-	]
+	],
+	"dart.flutterSdkPath": "/home/dmitry.detukov/opt/flutter/3.3.6-x64"
```
Comment (theanurin) [inline .vscode/settings.json:52]: Is this needed?

**Review by theanurin — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (thedetukov) [reply .vscode/settings.json:52]: Maybe not, I checked the program works without it:
"cSpell.words": [
		"Cupertino",
		"switchtile"
	]

Comment (theanurin) [reply .vscode/settings.json:52]: I mean `/home/dmitry.detukov/opt/flutter/3.3.6-x64`

Comment (thedetukov) [reply .vscode/settings.json:52]: No, i will fix it now

Comment (theanurin) [reply .vscode/settings.json]: Looks good )

**Review by theanurin — APPROVED** [review]

**Pull request merged**